.ci/update-github-page.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+set -e
+
+source ./.ci/util.sh
+
+if [[ -z $1 ]]; then
+  echo "release number is not set"
+  echo "usage: .ci/update-github-page.sh {release number}"
+  exit 1
+fi
+TARGET_RELEASE_NUM=$1
+
```

Comment:
Where is validation for GITHUB_TOKEN is set ?

Reply:
at that place only, I removed it to run a script to verify. Added again.